### PR TITLE
Upgrade kyaml to v0.10.20 in functions

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR is to upgrade kyaml to v0.10.19 in apply-setters, search-replace, upsert-resource, format, set-namespace, set-labels functions. The functions behavior doesn't change.

https://github.com/GoogleContainerTools/kpt/issues/2054

### CODE REVIEW

**Comment by phanimarupaka** [comment]
@mengqiy enforce-gatekeeper, ensure-name-substring, starlark, set-annotations functions are having some issues with update.

File: functions/go/apply-setters/go.mod [hunk 1]
```diff
@@ -5,5 +5,5 @@ go 1.15
 require (
 	github.com/stretchr/testify v1.6.1
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	sigs.k8s.io/kustomize/kyaml v0.10.10
+	sigs.k8s.io/kustomize/kyaml v0.10.19
```
Comment (mengqiy) [inline functions/go/apply-setters/go.mod:8]: Why not using v0.10.20?

File: functions/go/format/main.go [hunk 2]
```diff
@@ -3,7 +3,9 @@ package main
 import (
 	"fmt"
 	"os"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
```
Comment (mengqiy) [inline functions/go/format/main.go:6]: Can you please group std lib in one group and other lib in a separate group?

**Comment by mengqiy** [comment]
> enforce-gatekeeper, ensure-name-substring, starlark, set-annotations functions are having some issues with update.

I thought `set-annotations` is very similar to `set-labels`.
Can you please tell me a little bit more about the issue when updating these functions?

**Comment by mengqiy** [comment]
We are using 0.10.16 in https://kpt.dev/book/05-developing-functions/03-developing-in-Go?id=next-steps.
IMO we should use the same version of kyaml, then these function implementations can serve as examples.
I guess we should update our doc to use a newer kyaml.
WDYT?

**Comment by frankfarzan** [comment]
> We are using 0.10.16 in https://kpt.dev/book/05-developing-functions/03-developing-in-Go?id=next-steps.
> IMO we should use the same version of kyaml, then these function implementations can serve as examples.
> I guess we should update our doc to use a newer kyaml.
> WDYT?

Yes, that's part of https://github.com/GoogleContainerTools/kpt/issues/2054. @phanimarupaka can you include issues in the PR descriptions.

**Comment by frankfarzan** [comment]
this looks good to me, I'll let @mengqiy approve it.

File: functions/go/apply-setters/go.mod (hunk 1 above)
Comment (phanimarupaka) [reply functions/go/apply-setters/go.mod]: It released recently. Updated.

File: functions/go/format/main.go (hunk 2 above)
Comment (phanimarupaka) [reply functions/go/format/main.go]: Done.

**Review by mengqiy — APPROVED** [review]

**Pull request merged**